android-simple crashed

I compiled android-simple project with gradle, and install android-simple-debug-unaligned.apk to phone, it can not start:

```
06-17 22:58:39.428 D/AndroidRuntime( 7312): Shutting down VM
06-17 22:58:39.428 W/dalvikvm( 7312): threadid=1: thread exiting with uncaught exception (group=0x40e2e2a0)
06-17 22:58:39.438 E/AndroidRuntime( 7312): FATAL EXCEPTION: main
06-17 22:58:39.438 E/AndroidRuntime( 7312): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.example.dagger.simple/com.example.dagger.simple.ui.HomeActivity}: java.lang.IllegalArgumentException: No inject registered for members/com.example.dagger.simple.ui.HomeActivity. You must explicitly add it to the 'injects' option in one of your modules.
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2069)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2094)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.ActivityThread.access$600(ActivityThread.java:134)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1202)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.os.Handler.dispatchMessage(Handler.java:99)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.os.Looper.loop(Looper.java:137)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.ActivityThread.main(ActivityThread.java:4788)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at java.lang.reflect.Method.invokeNative(Native Method)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at java.lang.reflect.Method.invoke(Method.java:511)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:774)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:541)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at dalvik.system.NativeStart.main(Native Method)
06-17 22:58:39.438 E/AndroidRuntime( 7312): Caused by: java.lang.IllegalArgumentException: No inject registered for members/com.example.dagger.simple.ui.HomeActivity. You must explicitly add it to the 'injects' option in one of your modules.
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at dagger.ObjectGraph$DaggerObjectGraph.getInjectableTypeBinding(ObjectGraph.java:273)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at dagger.ObjectGraph$DaggerObjectGraph.inject(ObjectGraph.java:253)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at com.example.dagger.simple.DemoApplication.inject(DemoApplication.java:39)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at com.example.dagger.simple.DemoBaseActivity.onCreate(DemoBaseActivity.java:26)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at com.example.dagger.simple.ui.HomeActivity.onCreate(HomeActivity.java:27)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.Activity.performCreate(Activity.java:5031)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1082)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2033)
06-17 22:58:39.438 E/AndroidRuntime( 7312):     ... 11 more
06-17 22:58:39.451 W/ActivityManager(  309):   Force finishing activity com.example.dagger.simple/.ui.HomeActivity
```

The modification to compile with gradle is:
new     ：    build.gradle
rename：    AndroidManifest.xml -> src/main/AndroidManifest.xml
rename：    res/values/strings.xml -> src/main/res/values/strings.xml

The build.gradle content is:

```
jelly@jelly-VirtualDeepinRC:~/code/dagger/examples/android-simple$ cat build.gradle 
buildscript {
    repositories {
        mavenCentral()
    }  
    dependencies {
        classpath 'com.android.tools.build:gradle:0.4.2'
    }  
}

apply plugin: 'android'

repositories {
    mavenCentral()
} 
dependencies {
    compile 'com.squareup.dagger:dagger-compiler:1.0.1'
    compile 'com.squareup.dagger:dagger:1.0.1'
}

android {
    compileSdkVersion 17 
    buildToolsVersion "17.0.0"

    defaultConfig {
        minSdkVersion 9
        targetSdkVersion 16 
    }  
}

defaultTasks 'assemble'
```

--- a/examples/android-simple/src/main/java/com/example/dagger/simple/DemoApplication.java
+++ b/examples/android-simple/src/main/java/com/example/dagger/simple/DemoApplication.java
@@ -30,8 +30,9 @@ public class DemoApplication extends Application {
   }
 
   protected List<Object> getModules() {
-    return Arrays.<Object>asList(
-        new AndroidModule(this)
+    return Arrays.asList(
+        new AndroidModule(this),
+        new DemoModule()
     );
   }
 

